hetio/neo4j.py

```diff
@@ -5,21 +5,28 @@
 from operator import or_
 from functools import reduce
 
-import py2neo
-import py2neo.packages.httpstream
 import pandas
 from tqdm import tqdm
 
 import hetio.hetnet
 
-# Get py2neo version
-PY2NEO_VER = int(py2neo.__version__[0])
 
-# Avoid SocketError
-py2neo.packages.httpstream.http.socket_timeout = 1e8
+def import_py2neo():
+    """
+    Imports the py2neo library, checks its version, and sets the socket timeout if necessary
+    """
+    import py2neo
+    # Get py2neo version
+    PY2NEO_VER = int(py2neo.__version__[0])
+    if PY2NEO_VER < 4:
+        import py2neo.packages.httpstream
+        # Avoid SocketError
```

Comment:
can we add the following link to this comment `https://github.com/dhimmel/learn/issues/1`